core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -83,7 +83,7 @@ class AdminClient(val time: Time,
   }
 
   private def findAllBrokers(): List[Node] = {
-    val request = new MetadataRequest(List[String]())
+    val request = new MetadataRequest(List[String](), true)
```

Comment:
Should we just pass `null` instead of empty list (like in `RequestResponseTest`)?